src/player.js

```diff
@@ -1248,6 +1249,26 @@ export default class Player extends FakeEventTarget {
     return false;
   }
 
+  _applyTextTrackConfig(): void {
+    if (Utils.Object.getPropertyPath(this._config, 'text.forceCenter')) {
+      this.setTextDisplaySettings({position: 'auto', align: 'center', size: '100'});
+    }
```

Comment:
let's apply this after `textTrackDisplaySetting` as force will take precedence over any other `position`, `align` and `size`